src/os/filestore/FileJournal.cc

```diff
@@ -697,7 +697,7 @@ int FileJournal::read_header(header_t *hdr) const
   if (r < 0) {
     int err = errno;
     dout(0) << "read_header got " << cpp_strerror(err) << dendl;
-    return -err;
```

Comment:
instead of fixing in `read_header()`, can we just do this in `FileJournal::open()` to be consistent the rest of `FileJournal` class where `read_header()` is called?